clients/include/testing_asum.hpp

```diff
@@ -37,7 +37,7 @@ inline void testname_asum(const Arguments& arg, std::string& name)
 }
 
 template <typename T>
-inline hipblasStatus_t testing_asum(const Arguments& arg)
+inline void testing_asum(const Arguments& arg)
```

Comment:
Was never sure why these were inline (pre-existing), compiler may be ignoring anyway